testnet/stacks-node/conf/local-leader-conf.toml

```diff
@@ -17,19 +17,124 @@ commit_anchor_block_within = 20_000
+address = "ST2VZ8CH12MGJXVAF54PKE3D7K4HVRVEJX11WBGXR"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 1b5fddefb32d2bf3ea9ca1a0b9c8c5d8e0e82c101e94f30a417ac64ae402218701
+address = "ST33M0BJQN169PDVFVT7Z7T83V86903A32KBKQYZ5"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 5a1e7817bc62ce5abea4f27b8002c7de02055bcf04a6e567a065ab06842fe34e01
+address = "ST12WEE0Y5HSSVRM8XS4TCEQVPYCVKSW8SVNNGYME"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 4bfcb79c3383519386b807eda574324f146044effc3cbb53e5648e05d820e8e701
+address = "ST3VEP0E2HJBTX6HY5TB9CC3G4CQNKF4RCNW7WWDS"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 1a1453957427a0eaa0f064845750ab0930abea764d060e37a0a90065e0cfab6a01
+address = "ST15NWBK91FTCGKQZ6JGDYJ5H12FRRSP3CFFNE5P6"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: b6495e7953b4e61baa6377e424de6d4d2723bb0a3006a5189c4eb3acce739a7601
+address = "ST1D0XTBR7WVNSYBJ7M26XSJAXMDJGJQKNEXAM6JH"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 5ab899130953826ac53235a633eef111c1e213ad8f62a2c8485317fc21b8eef801
+address = "ST14KHGDNAX1R5Q7V4P0S3KSQRWZ4P10W0DH7V170"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 29643a52a7c4a054878601dc5e6dc93ef6db0dcae059e91b2657d495efaafddd01
+address = "ST17H7G0D8ZDQWCSAPEMJD7ZHRESWKFEDCZ039G7S"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 72badaf3be5ceaa1b453fcf04eed27711a3ffed95fbe79df06eedc11ae185b9801
+address = "ST3Z4WNZEHJBNRBKY7X6ENXRH583YS6FVH4S89SCE"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 0d75216d199f90968fd8c779c9e7ace6a8979a9f9a246a97defc6e7a5eebd0a601
+address = "ST4TKY7QFS5C3X23628PQGT3910ZA53HSWZQ76MG"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: d4c3068518da81fdf0c5b7bcdc10a6eceeb1b1ee9b711d038bed6eda0a6ab22501
+address = "ST37VVF50JVE2QJZR4DXP1Y9EDYZC2CMZJXMJ2SHQ"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 9ba45056d1bb3bec571ad0901f143e204a7dc42c804554baffcc45d016726d9601
+address = "ST8931RAJX6KJCNY9GK2B9NEJT7Y4SAC4MAYC3KR"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 6df704626c69738e0c45228c440884ab6fd91bca6a80b7e8fcb551bd7c19935f01
+address = "STPB5Z0E1G3AA264YHFCYD3Z2R5TEYCFFXMKXTC1"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: 1f5c6a6fe9616c071aa412442524023aa545cd20d0a83d7483a1ba6fda7a079601
+address = "ST3SY2ZJT26C0T948WWMG3XKGXXJXTW46XYPZG28H"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: bc5710d0548ac0a44903c29841c737f98f031147f071b372550029c3c7abc50201
+address = "ST2XKCRMJWF07NYRJP6GRATWC4G1D4ZJZ6GSS0WX0"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: f96349d0e4261abc7a5ef6b7d32c0f367612982c0cd995478defcd1e96e57ce201
+address = "ST1D663PR9RPVZH9X3JRFM5Z90ZR2Y91C5XAJTN3H"
+amount = 100000000000000
+[[mstx_balance]]
+# Private key: b4dc4945b57c9649ae2ef51aba46ad24f1ec45721969b096a4a9eabfff5e58df01
+address = "ST289WH00WPBAG852K4FRCBNFV7HQ76SZKBHQZ427"
+amount = 100000000000000
+
```

Comment:
If we deploy this, we'll need to reset the chain (since these addresses will not be included in the genesis block)

Reply:
This is just `local-leader-conf.toml`, not the `neon-leader-conf.toml`